smdebug/core/state_store.py

```diff
@@ -33,6 +33,7 @@ def _get_checkpoint_files_in_dir(self, cp_dir):
                     file != METADATA_FILENAME
                     and file != METADATA_FILENAME_S3_UPLOADED
                     and "sagemaker-uploaded" not in file
+                    and "sagemaker-uploading" not in file
```

Comment:
nit: This won't scale well. Can we keep track of a list of files not to append and then do something like:

```
if all([metadata_file not in file for metadata_file in metadata_files]) 
```